includes/templates/bootstrap/css/stylesheet_zca_colors.php

```diff
@@ -241,4 +246,10 @@
 .productFreePrice {
 	color: <?php echo ZCA_BODY_PRODUCTS_FREE_COLOR; ?>;
 	}
+	
+
+.legalCopyright, .legalCopyright a {color: <?php echo ZCA_FOOTER_EZPAGE_LINK_COLOR; ?>; text-decoration: none}
```

Comment:
I think the `text-decoration` part can be replaced with `text-decoration-none` in the template-file. I'll remove it from here.

Reply:
OK, we'll leave the file changes and name as-is.  No sense in mangling any upgrades with under-the-cover changes.